### DISCUSSION TITLE
如何手动将 Nike 中国站打包下载的TCX数据手动导入现有的工程里面呢

### DISCUSSION THREAD

**Discussion by cigar** [yihong0618/running_page#410] [Q&A] [answered]
你好，去年Nike Running Club的数据我从官网下载下来以后，它的格式是一个ZIP包，解压以后就是一个TCX后缀的数据，请问我如何将2千多份数据手动放入我们的工程里面呢？

**Comment by cigar** [comment]
我当前把Nike的数据解压后的tcx文件放入TCX_OUT的目录，运行功能后，这个数据为能体现在工程的数据里面

**Reply by yihong0618** [reply, marked as answer]
弄一个 clean 的项目，然后使用 tcx_sync
